### PR TITLE
Path matching: transform Fabric path to Skipper path

### PR DESCRIPTION
**Pull request by marcinzaremba**
Signed-off-by: Marcin Zaremba <marcin.zaremba@zalando.de>

### CODE REVIEW

File: fabric/path_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package fabric
+
+import "testing"
+
+func TestFabricPathToPredicate(t *testing.T) {
```
Comment (marcinzaremba) [inline fabric/path_test.go:5]: This test represents current Fabric behaviors as is. @szuecs Are there current behaviours that you would like to change/deprecate?

File: fabric/path.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		{regexp.MustCompile(slash + star + wildcardName + end), `/:$name`},
+		{regexp.MustCompile(curlyOpen + wildcardName + curlyClose), `:$name`},
+		{regexp.MustCompile(slash + star + slash), `/:id/`},
+		{regexp.MustCompile(slash + star + end), `/:id`},
+	}
+)
+
+// fabricPathToPredicate takes a Fabric path string and transforms it an equivalent Skipper Path* predicate
+func fabricPathToPredicate(fp string) *eskip.Predicate {
+	if fp == "/**" {
```
Comment (szuecs) [inline fabric/path.go:31]: What about `/foo/**` or this case does not exist?

File: fabric/path_test.go (hunk 1 above)
Comment (szuecs) [reply fabric/path_test.go:5]: I don't know, I think it should be the same. If you have a reason to change it let me know, but I guess there is no reason.

File: fabric/path.go (hunk 2 above)
Comment (marcinzaremba) [reply fabric/path.go:31]: In original version (Scala one) only `/**` is represented as `PathSubtree`. Rest just lands to `Path`. I do not see a reason why just `ends with /**` is not also transformed to `PathSubtree`. Maybe we can ask original Fabric guys.

Comment (marcinzaremba) [reply fabric/path.go:31]: Asked in our Fabric and Skipper channel.

File: fabric/path_test.go (hunk 1 above)
Comment (marcinzaremba) [reply fabric/path_test.go:5]: I mainly think about poor documented `*named-match` cases. It only appeared to me by reading code and tests, not the docs. I plan to check the adoption of those.

**Comment by marcinzaremba** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Pull request merged**